pkg/routing/selector/interfaces.go

```diff
@@ -24,20 +24,22 @@ func CreateNodeSelector(conf *config.Config) (NodeSelector, error) {
 	case "cpuload":
 		return &CPULoadSelector{
 			CPULoadLimit: conf.NodeSelector.CPULoadLimit,
+			SortBy: conf.NodeSelector.SortBy,
 		}, nil
 	case "sysload":
 		return &SystemLoadSelector{
 			SysloadLimit: conf.NodeSelector.SysloadLimit,
+			SortBy: conf.NodeSelector.SortBy,
 		}, nil
 	case "regionaware":
-		s, err := NewRegionAwareSelector(conf.Region, conf.NodeSelector.Regions)
+		s, err := NewRegionAwareSelector(conf.Region, conf.NodeSelector.Regions, conf.NodeSelector.SortBy)
 		if err != nil {
 			return nil, err
 		}
 		s.SysloadLimit = conf.NodeSelector.SysloadLimit
 		return s, nil
 	case "random":
-		return &RandomSelector{}, nil
+		return &RandomSelector{conf.NodeSelector.SortBy}, nil
```

Comment:
if the `Random` selector is to take in a sort field that's not random, this would be pretty confusing as the name becomes meaningless. wdyt about leaving this one alone?

Reply:
I thought about this, but at the moment "random" is the only one that would expose all nodes without some kind of limit. Ideally, the "random" selector would actually be called something like "all" or "any", but I didn't want to change something that was the default.

I'm happy to remove this, as I agree it's a little confusing, but I could see a use case for it.

Reply:
how about let's do this: remove it for now to eliminate confusion. Then we can always add a new node selector for the use cases that desire it.

I don't think anyone should be using the random selector FWIW, just wanting to keep an eye on things that introduce increased cognitive load.

Reply:
Actually, what do you think about renaming the `random` selector to `any` and keeping `random` in interfaces pointing to `any` with a deprecation warning? Looks like that's a trivial code change that I could add here as a separate commit if it sounds good.

Reply:
I updated this PR to rename `random`. Personally, I prefer this approach. However, I opened #612 if you prefer to go with sort by not applying to the random selector,

Reply:
looks good to me!